src/components/input/select.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+import DropDown from 'react-native-paper-dropdown';
+import { AppColor } from 'src/common/enums';
+
+import styles from './styles';
+import theme from './theme';
+
+type Props = Omit<
+  React.ComponentPropsWithoutRef<typeof DropDown>,
+  'visible' | 'onDismiss' | 'showDropDown'
+>;
+
+const Select: React.FC<Props> = dropDownProps => {
+  const [open, setOpen] = useState<boolean>(false);
+
+  const show = () => setOpen(true);
+  const hide = () => setOpen(false);
+
```

Comment:
Component methods must be named via handle

```suggestion
  const handleShow = () => setOpen(true);
  
  const handleHide = () => setOpen(false);

```